lua-cjson-2.1.0.6-1.rockspec

```diff
@@ -1,8 +1,9 @@
 package = "lua-cjson"
-version = "2.1devel-1"
+version = "2.1.0.6-1"
```

Comment:
Why the `-1` suffix in the version number?

Reply:
luarocks will check if the `version` follows `[%w.]+-[%d]+`, so we need the `-1` suffix here.